kong/types.go

```diff
@@ -127,6 +127,7 @@ type ActiveHealthcheck struct {
 // +k8s:deepcopy-gen=true
 type PassiveHealthcheck struct {
 	Healthy   *Healthy   `json:"healthy,omitempty" yaml:"healthy,omitempty"`
+	Type      *string    `json:"type,omitempty" yaml:"type,omitempty"`
```

Comment:
Can you please add this to `Active` too?
Also, can you please add an integration test (or add to an existing one) for this field?

Reply:
This looks to already be available on `Active`. I added a test that captures the functionality I'm introducing here